docker/install/ubuntu_install_ethosn_driver_stack.sh

```diff
@@ -54,4 +48,4 @@ cd "$tmpdir"
 git clone --branch "$repo_revision" "$repo_url" "$repo_dir"
 
 cd "$repo_dir"/driver
-scons install_prefix="$install_path" install
+scons werror=False install_prefix="$install_path" install
```

Comment:
Do you mind opening an issue to report what is the error that makes this script now need `werror=False`?

I understand it will have some output, but can you add that output in the issue, so that somebody can pick that up to investigate?

Reply:
I got error like "error: loop variable 'pair' creates a copy from type 'const std::pair<const unsigned int, std::pair<long unsigned int, long unsigned int> >' [-Werror=range-loop-construct] for (const auto pair : passAgentIdRanges)", it should be a warning, so I added werror=False in scons install for that, I think the ideal solution is to modify the code in https://github.com/Arm-software/ethos-n-driver-stack to fix this warning. More details about the error info. please refer to the log https://ci.tlcpack.ai/blue/rest/organizations/jenkins/pipelines/tvm-docker/branches/PR-14635/runs/5/nodes/73/steps/129/log/?start=0